From ece1f735a7423a4ed06bb422dca4dd6094807694 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 19 Oct 2023 20:23:57 +0200
Subject: [PATCH] command-resource-handlers.json: Switch to new variants of
 browser, certificate, maps and messaging apps

To use the available apps instead of the not available com.palm ones.
Drop commands and handlers unlikely to ever return (Sprint).

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/conf/command-resource-handlers.json | 38 ++++++++++-------------
 1 file changed, 17 insertions(+), 21 deletions(-)

diff --git a/files/conf/command-resource-handlers.json b/files/conf/command-resource-handlers.json
index 26d00f9..b30839f 100644
--- a/files/conf/command-resource-handlers.json
+++ b/files/conf/command-resource-handlers.json
@@ -8,17 +8,17 @@
 	
 	"resources":
 	[
-		{ "extn":"htm", "mime":"text/html", "appId":"com.palm.app.browser", "streamable":true },
-		{ "extn":"html", "mime":"text/html", "appId":"com.palm.app.browser", "streamable":true },
+		{ "extn":"htm", "mime":"text/html", "appId":"com.webos.app.enactbrowser", "streamable":true },
+		{ "extn":"html", "mime":"text/html", "appId":"com.webos.app.enactbrowser", "streamable":true },
 		{ "extn":"vcf", "mime":"text/x-vcard", "appId":"com.palm.app.contacts", "streamable":false },
 
-		{ "extn":"pem", "mime":"application/pem", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"p12", "mime":"application/p12", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"pfx", "mime":"application/pfx", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"der", "mime":"application/der", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"crt", "mime":"application/crt", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"cer", "mime":"application/cer", "appId":"com.palm.app.certificate", "streamable":false },
-		{ "extn":"crl", "mime":"application/crl", "appId":"com.palm.app.certificate", "streamable":false },
+		{ "extn":"pem", "mime":"application/pem", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"p12", "mime":"application/p12", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"pfx", "mime":"application/pfx", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"der", "mime":"application/der", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"crt", "mime":"application/crt", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"cer", "mime":"application/cer", "appId":"org.webosports.app.settings", "streamable":false },
+		{ "extn":"crl", "mime":"application/crl", "appId":"org.webosports.app.settings", "streamable":false },
 
 		{ "extn":"mov", "mime":"video/quicktime", "appId":"com.palm.app.videoplayer", "streamable":true },
 		{ "extn":"mp4", "mime":"video/mp4-generic", "appId":"com.palm.app.videoplayer", "streamable":true },
@@ -94,21 +94,17 @@
 	"commands":
 	[
 		{ "url":"^mailto:", "appId":"com.palm.app.email" },
-		{ "url":"^im:", "appId":"com.palm.app.messaging" },
-		{ "url":"^sms:", "appId":"com.palm.app.messaging" },
+		{ "url":"^im:", "appId":"org.webosports.app.messaging" },
+		{ "url":"^sms:", "appId":"org.webosports.app.messaging" },
 		{ "url":"^contact:", "appId":"com.palm.app.contacts" },
-		{ "url":"^chatWith:", "appId":"com.palm.app.messaging" },
+		{ "url":"^chatWith:", "appId":"org.webosports.app.messaging" },
 		{ "url":"^opencontact:", "appId":"com.palm.app.contacts" },
-		{ "url":"^mojave:", "appId":"com.palm.app.mojave" },
-		{ "url":"^https?:", "appId":"com.palm.app.browser"},
-		{ "url":"^data:", "appId":"com.palm.app.browser"},
+		{ "url":"^https?:", "appId":"com.webos.app.enactbrowser"},
+		{ "url":"^data:", "appId":"com.webos.app.enactbrowser"},
 		{ "url":"^tel:", "appId":"com.palm.app.phone"},
 		{ "url":"^mobi:", "appId":"com.palm.app.videoplayer"},
-		{ "url":"^mapto:", "appId":"com.palm.app.maps"},
-		{ "url":"^maploc:", "appId":"com.palm.app.maps"},
-		{ "url":"^rtsp[tu]?:", "appId":"com.palm.app.videoplayer"},
-		{ "url":"^sprinttv:", "appId":"com.palm.mobitv.app.sprinttv" },
-		{ "url":"^sprint-music:", "appId":"com.palm.sprintmusicplus" },
-		{ "url":"^ypc:", "appId":"com.palm.app.ypmobile" }
+		{ "url":"^mapto:", "appId":"org.webosports.app.maps"},
+		{ "url":"^maploc:", "appId":"org.webosports.app.maps"},
+		{ "url":"^rtsp[tu]?:", "appId":"com.palm.app.videoplayer"}
 	]
 }
